src/components/CategoryReviews.jsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react";
+import { fetchReviewsByCategory } from "../utils/utils";
+import { Link, useParams } from "react-router-dom";
+
+
+
+const CategoryReviews = () => {
```

Comment:
This could also be acheived in the other Reviews component. This component also doesn't allow you to change the filter again